tests/mdev_configuration_allocation_test.go

```diff
@@ -144,7 +150,7 @@ var _ = Describe("[Serial][sig-compute]MediatedDevices", func() {
 			tests.UpdateKubeVirtConfigValueAndWait(config)
 
 			By("Verifying that an expected amount of devices has been created")
-			checkAllMDEVCreated(desiredMdevTypeName, expectedInstancesNum)
+			Eventually(checkAllMDEVCreated(desiredMdevTypeName, expectedInstancesNum), 3*time.Minute, 15*time.Second).Should(BeInPhase(k8sv1.PodSucceeded))
```

Comment:
wow this is too long.

Would you mind adding some newlines here ? 

Maybe 

```golang
Eventually(
    checkAllMDEVCreated(desiredMdevTypeName, expectedInstancesNum), 
    3*time.Minute,
    15*time.Second).Should(
        BeInPhase(k8sv1.PodSucceeded))
```

Feel free to ignore - I mostly just don't like to scroll horizontally .

Reply:
I will change this if another change is required.